core/src/actors/paths.rs

```diff
@@ -310,35 +310,51 @@ impl ActorPath {
     /// Same as [tell](ActorPath::tell), but serialises eagerly into a Pooled buffer (pre-allocated and bounded)
     pub fn tell_serialised<CD, B>(&self, m: B, from: &CD) -> Result<(), SerError>
     where
-        CD: ComponentDefinition + Sized + 'static,
+        CD: ComponentTraits + ComponentLifecycle,
```

Comment:
Would it still compile with the ComponentDefinition one? I still see it exists in the codebase, so I am bit confused between ComponentDefinition vs. ComponentTraits + ComponentLifecycle.

Reply:
I simply got tired of writing a long list of traits every time, so I did the fake trait alias thingy. But adding `ComponentLifecycle` would have been both unnecessary in some places and lead to a dependency cycle, so it's separate. The old long version should still work in places where `ComponentLifecycle` isn't needed, otherwise you have to add it.